client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -175,19 +197,124 @@ const getEmailForwardMenuItems = ( { dispatch, mailbox, translate } ) => {
 	];
 };
 
+const RemoveTitanMailboxConfirmationDialog = ( { mailbox, visible, setVisible } ) => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const emailAddress = getEmailAddress( mailbox );
+
+	const noticeDuration = 7000;
+
+	const errorMessage = errorNotice(
+		translate(
+			'There was an error removing {{strong}}%(emailAddress)s{{/strong}} from your account',
+			{
+				comment: '%(emailAddress)s is the email address associated to the mailbox being deleted',
```

Comment:
Nit: it might be simpler to express this as:
```suggestion
				comment: '%(emailAddress)s is the email address for the mailbox being deleted',
```